src/scripts/main.js

```diff
@@ -355,6 +355,21 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
```

Comment:
It's a good practice to check if the element exists before using it. This helps to avoid potential errors in the future. You can do this by adding a simple condition like `if (!dashboard) return;`